deploy/1.8+/metrics-server-deployment.yaml

```diff
@@ -42,6 +44,7 @@ spec:
           runAsNonRoot: true
           runAsUser: 1000
         imagePullPolicy: Always
+        hostNetwork: true
```

Comment:
Running in host network can help in some Kubernetes distributions, but currently I'm not convinced that this is a good default